tarodb/assets_store.go

```diff
@@ -1470,12 +1480,8 @@ func (a *AssetStore) ConfirmParcelDelivery(ctx context.Context,
 			return err
 		}
 
-		// Before we conclude we'll remove the old proofs we were
-		// staging on disk.
-		err = q.DeleteSpendProofs(ctx, assetTransfer.TransferID)
-		if err != nil {
-			return err
-		}
+		// Keep the old proofs as a reference for when we list past
+		// transfers.
```

Comment:
Still getting through the the diff, but as is we might end up storing _slightly_ redundant data, since these are the proof suffixes (new state transitions). This would only really be for the sender though, since the receiver's aren't stored in the DB though.